source/ui/Image.js

```diff
@@ -174,12 +186,17 @@
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
 			if (this.sizing) {
-				this.applyStyle('background-image', src ? 'url(' + enyo.path.rewrite(src) + ')' : 'none');
+				var placeholder = this.placeholder ? ', url(\'' + enyo.path.rewrite(this.placeholder) + '\')' : '';
+				var url = 'url(\'' + enyo.path.rewrite(src) + '\')' + placeholder + ';';
```

Comment:
I think we need to guard against a falsy value such as an empty string here, otherwise we'll set the `background-image` to `url('')`, which results in an error as we are trying to set the background-image to the current page's URL:

```
Resource interpreted as Image but transferred with MIME type text/html
```